README.md

```diff
@@ -68,42 +67,32 @@ Encoder as input require raw YUV file (422, 444), 10-bit or more.
 
 Displaying help:
 
-    $> oapv_app_enc --help
```

Comment:
I think this needs to be here.
'$>' means console environment.

Reply:
All sections with console input are wrapped into formatting tags so it's visible that is console environment.
Adding **$>** symbols is redundant and also generate problem when copying console section to terminal, because those symbols need to be removed manually before code execution. 
I think that we need to unify this everywhere and remove redundant console markings.